### ISSUE TITLE
Diamond topology not supported

### ISSUE THREAD

**Issue by alegaballo** [mhogan26/Portfolio_Path_Management#1]
It seems that there is no support for the diamond topology in [VirtualNetworkGraph.py](https://github.com/mhogan26/Portfolio_Path_Management/blob/master/VirtualNetworkGraph.py#L236). 
In the sequence of check starting at line 236 diamond is missing, am I wrong?

**Comment by mhogan26** [comment]
You're correct; I intended the diamond topology to be a simple 4-node network, so it was quicker to create the diamond.out file by hand instead of using VirutalNetworkGraph.py
